src/components/pages/dashboard/panels/kpis.js

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import React, { Component } from 'react';
+import { Observable } from 'rxjs';
+
+import { Indicator } from 'components/shared';
+import {
+  Panel,
+  PanelHeader,
+  PanelContent,
+  PanelOverlay
+} from 'components/pages/dashboard/panel';
```

Comment:
Seems like these will be imported a lot together. Should it have an index.js to simplify?

Reply:
It is being imported from an index.js file